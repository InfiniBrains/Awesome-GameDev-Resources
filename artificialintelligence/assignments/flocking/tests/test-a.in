1.000 0.000 0.000 0.000 1.000 0.000 0.000 2
0.000 0.500 0.000 0.000
0.000 -0.500 0.000 0.000
0.125
