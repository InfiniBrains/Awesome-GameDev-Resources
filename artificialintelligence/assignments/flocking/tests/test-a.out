0.000 0.484 0.000 -0.125
0.000 -0.484 0.000 0.125
